docs/source/index.rst

```diff
@@ -138,6 +138,7 @@ Please see our `contributing guidelines <https://github.com/cleanlab/cleanlab/bl
    Token Classification (text) <tutorials/token_classification>
    Predicted Probabilities via Cross Validation <tutorials/pred_probs_cross_val>
    FAQ <tutorials/faq>
+   Datalab Tutorials <tutorials/datalab/index>
```

Comment:
```suggestion
   Datalab Tutorials <tutorials/datalab/index>
   Predicted Probabilities via Cross Validation <tutorials/pred_probs_cross_val>
   FAQ <tutorials/faq>
```

Reply:
Consistently order where datalab tutorials appear in the side bars.
My personal pref is for them to appear higher up in the side bar, but it should be at least this high